src/include/sof/lib/dai.h

```diff
@@ -22,6 +22,7 @@
 #include <errno.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <sof/audio/component.h> /* COMP_TRIGGER_* macros */
```

Comment:
We either comment why each of includes is needed or don't make comment like this at all. I'd remove that comment to be consistent.

Reply:
Sure

Reply:
Thanks